wiki/History_of_osu!/1/en.md

```diff
@@ -110,9 +127,12 @@ No joke, **Bancho went live this month, and so did the user panels of Extended C
 May
 ---
 
-![[Taiko](Taiko "wikilink") mod](Taiko.png "fig:Taiko mod") The **[Taiko](Taiko "wikilink") Mode** first saw the light of day in this month, as well as another major addition, **.osb storyboarding script support. Multi-part sliders and private messaging in chat were also made available.**
+![Taiko mod](img/Taiko.png "fig:Taiko mod")
```

Comment:
use original size and remove `fig:`
![taiko](https://cloud.githubusercontent.com/assets/14264992/25783409/6abda658-3321-11e7-9c84-de5b6b0b2929.png)